services/visual_recognition/v1.js

```diff
@@ -0,0 +1,237 @@
+module.exports = function(RED) {
+
+	var cfEnv = require("cfenv");
+
+	var watson = require('watson-developer-cloud');
+	var temp = require('temp');
+	temp.track();
+	
+	var fs = require('fs');
+	var fileType = require('file-type');
+	var request = require('request');
+
+	var appEnv	= cfEnv.getAppEnv();
+		
+	var visual = [];
+	for (var i in appEnv.services) {
+		if (i.match(/^(visual_recognition)/i)) {
+			visual = visual.concat(appEnv.services[i].map(function(v) {
+				return {
+					name: v.name, 
+					label: v.label, 
+					username: v.credentials.username,
+					password: v.credentials.password,
+					};
+			}));
+		}
+	}
+
+	RED.httpAdmin.get('/watson-visual-recognition/vcap', function(req,res) {
+		res.send(JSON.stringify(visual));
+	});
+	
+	RED.httpAdmin.get('/watson-visual-recognition/list/:service', function(req,res) {
+
+		var username,password;
+		var service = req.params.service;
+		
+		for (var i2=0; i2 < visual.length; i2++) {
+			if (visual[i2].name===service) {
+				username = visual[i2].username;
+				password = visual[i2].password;
+			}
+		}
+		
+		var visual_recognition = watson.visual_recognition({
+			username: username,
+			password: password,
+			version: 'v2-beta',
+			version_date: '2015-12-02'
+		});
+			
+		visual_recognition.listClassifiers({},
+			function(err, response) {
+				if (err) 
+					res.send(err);
+			 	else
+					res.send(JSON.stringify(response));
+			}
+		);
+	});
+
+	function VisualUtilNode(config) {
+		
+		RED.nodes.createNode(this,config);
+
+		this.name = config.name;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 0.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2400711164/issues/source?bid=3225115&fileBranchId=3232279#l66)